memory/lessons.md

```
@@ -357,6 +357,36 @@
 
 ---
 
+## Category: Supabase Edge Functions (continued)
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)